engine/src/main/java/org/terasology/rendering/nui/layers/ingame/metrics/DebugOverlay.java

```diff
@@ -37,10 +36,12 @@
 import org.terasology.world.biomes.Biome;
 import org.terasology.world.biomes.BiomeManager;
 
-import java.util.List;
 import java.util.Locale;
 
 /**
+ * Displays and changes metrics modes, according to incoming actions from DebugControlSystem.
+ *
+ * DebugMetricsSystem provides available metrics modes to DebugOverlay.
```

Comment:
I'd write:

/**
Displays the content of the MetricsMode instances provided by the DebugMetricsSystem. 

See the toggleMetricsMode() method to iterate through the MetricsMode instances available for display.
*/